common/cross-profiles/i686.sh

```diff
@@ -8,3 +8,5 @@ XBPS_CROSS_CXXFLAGS="$XBPS_CROSS_CFLAGS"
 XBPS_CROSS_FFLAGS="$XBPS_CROSS_CFLAGS"
 XBPS_CROSS_RUSTFLAGS="--sysroot=${XBPS_CROSS_BASE}/usr"
 XBPS_CROSS_RUST_TARGET="i686-unknown-linux-gnu"
+XBPS_CROSS_ZIG_TARGET="i686-linux-gnu"
+XBPS_CROSS_ZIG_CPU="baseline"
```

Comment:
Nit: you can enable sse2 if you want, if it isn't by default.

Reply:
done!